components/ProductImage/index.jsx

```diff
@@ -108,6 +111,11 @@ class ProductImage extends Component {
       );
     }
 
+    if (this.props.optimizedImages) {
+      return (
+        <Picture sources={this.props.optimizedImages} alt={this.props.alt} square />
```

Comment:
`onError={this.imageLoadingFailed}` is not used in case of the new picture element here. 
Example: https://images.shopgate.services/440x440/filters:format(jpeg):quality(60)/https://img-cdn.shopgate.com/30916/1/052bb2c0c8851bb888ce23b0c22e081e562e6ab6ee56ef434f2ff6783eb12a48%3Fw%3D440%26h%3D440%26q%3D70%26zc%3Dresize%26fillc%3DFFFFFF?Expires=1536742519&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9pbWFnZXMuc2hvcGdhdGUuc2VydmljZXMvNDQweDQ0MC9maWx0ZXJzOmZvcm1hdChqcGVnKTpxdWFsaXR5KDYwKS9odHRwczovL2ltZy1jZG4uc2hvcGdhdGUuY29tLzMwOTE2LzEvMDUyYmIyYzBjODg1MWJiODg4Y2UyM2IwYzIyZTA4MWU1NjJlNmFiNmVlNTZlZjQzNGYyZmY2NzgzZWIxMmE0OCUzRnclM0Q0NDAlMjZoJTNENDQwJTI2cSUzRDcwJTI2emMlM0RyZXNpemUlMjZmaWxsYyUzREZGRkZGRiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTUzNjc0MjUxOX19fV19&Signature=Rr96iC9KPVGVuxAWF1EOhqpPQULBU3XTiYBtBOTcAgatJHqtPxo1-P0EGjbEmtCFf5PF2Am0nfcDpWh~PXPMYU2FdVaUHkhxmotpat5v9SxqfAgJzp9Yljmdsg1IEGSIVx1iSI07uM~KnNcb3QpJLV4R5tUUxOq4eicS5yE1S~JQK-bGXc8ImD1Dzu4MqvLw~jz3gHVZ1~0yD~UkuA9GLWpAzDZ8Bb10GLlJbg3MMprv1nhwJusVw4Mk8mjkmbcUuGi0AlEyGQjmXZgEhcyKnC~BkPuycDtxDNeHwEwZzH0P-VEEJOnvGgEVwoMI3cYiYJcOZAe8skfRNdYCuIEctQ__&Key-Pair-Id=APKAINMNDAWPE3XM2H7Q